### ISSUE TITLE
book updates

### ISSUE THREAD

**Issue by fbstj** [PaleoCrafter/book-search#2]
what happens when we get a new edition of a book such as the 10th aniversary copies or the WoR corrections. it would be really awesome if we could somehow keep hold of the old versions for advanced usage (for trivia searches and such things)

**Comment by PaleoCrafter** [comment]
The search doesn't (yet?) have a concept of "editions". It only knows books, which may contain whatever they want. Books may be sorted into series/categories.

A potential feature that could help with keeping around old editions while not cluttering the search with multiple results from different editions would be a setting for which series/books are included in the default filter.

**Comment by fbstj** [comment]
yeah I was possibly thinking of a too complex entire system. a way to keep previous versions around for advanced searches would probably be good enough

**Comment by jofwu** [comment]
Wrote all of this up as a separate Issue before thinking to see if the idea had already been proposed. XD Oops. I'm just going to copy and paste:

> I'm wondering if it would be useful to keep a record of old book versions. I can see a benefit to having the ability to reference older revisions every once in a while. If we get a version of this software to Peter, I can definitely imagine that might be useful to him.
> 
> Perhaps the revision can be pulled from ebooks somehow... But I guess it probably needs to be manually entered? I think Eric linked a crowdsourced site in #staff with all of Brandon's different book versions that exist. Probably can look at that to determine the best method for naming/numbering versions.
> 
> When you add a new version it would simply need to keep a record of the previous. I guess you'd want a checkbox when uploading a new version to say "make this the new primary version".
> 
> The idea would be that search defaults only to the "primary" versions of each book. I imagine the best way to uncover the rest would be a toggle somewhere telling it to include old versions. On search result pages it could treat these as separate books, with the version name/date/whatever appended to the book title??? But probably better if it added another level of dropdowns between "books" and "chapters" so it's Book > Version > Chapter.
